tools/git-utils.sh

```diff
@@ -0,0 +1,80 @@
+# doesn't do anything on its own.  must be sourced.
```

Comment:
I did this and renamed `CLONE_TARGET` to `SUBDIR`.  This is the name that the pixel tests were using and I like it better because we're not always cloning (eg: in the dist/ case).